packages/laconia-batch/src/recurse.js

```diff
@@ -1,12 +1,18 @@
-const invoker = require("@laconia/invoker");
+const { InvokeCommand } = require("@aws-sdk/client-lambda");
 const isplainobject = require("lodash.isplainobject");
 const _ = { isPlainObject: isplainobject };
 
-module.exports = ({ event, context, $lambda }) => (payload = {}) => {
+module.exports = laconiaContext => async (payload = {}) => {
   if (!_.isPlainObject(payload)) {
     throw new Error("Payload must be an object");
   }
-  return invoker(context.functionName, $lambda).fireAndForget(
-    Object.assign({}, event, payload)
+
+  const { context, event, $lambda } = laconiaContext;
```

Comment:
question: do we need to pass in a default `new LambdaClient()` here, because `$lambda` doesn't get populated anymore by default in `laconia-core`?